### ISSUE TITLE
Quitting the Quick view mode reset the cursor/point at the beggining of the Ztree buffer and not to the current file.

### ISSUE THREAD

**Issue by 4goodapp** [fourier/ztree#69]
when you place the point at a file and use **v** to view it quickly, when you quit with q, the cursor is not placed back to the file but to beggining of the ztree buffer.

**Comment by fourier** [comment]
Good catch. Unfortunately I was not able to find a quick way to fix it yet..

**Comment by DrChr** [comment]
Hi,
I just started having a look at ztree-dir and it seems _very_ neat, but I ran in to an annoying UX issue.

This issue might belong here, or maybe I should create a separate one - please just let me know in that case.

I'm using ztree that I just installed from melpa (20191108.2234), please let me know if I should test with another version.

Here's how to reproduce:
* Have Emacs show a single buffer, i.e. C-x 1
* M-x ztree-dir     (for some directory)
* Move the cursor down a bit, e.g. do PgDn, and press Enter  (ztree-perform-action)

This makes Emacs show a second window, where the second window shows the selected file. However, the cursor in the window for the ztree-dir has now moved (up a number of lines). This is very annoying.

I'm able to reproduce this with a 'clean' Emacs setup, so it shouldn't depend on my local configuration. I.e. I also reproduced this issue after having started Emacs using the following command:
$ emacs -q --eval "(package-initialize)"

**Comment by fourier** [comment]
@DrChr I cannot reproduce this issue anymore, If you can please create a separate issue.

**Comment by DrChr** [comment]
I've now created a separate issue, #78
